edward-elgar-business-and-social-sciences.csl

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Edward Elgar Publishing - Business &amp; Social Sciences</title>
+    <id>http://www.zotero.org/styles/edward-elgar-business-and-social-sciences</id>
+    <link href="http://www.zotero.org/styles/edward-elgar-business-and-social-sciences" rel="self"/>
+    <link href="http://www.zotero.org/styles/british-journal-of-industrial-relations" rel="template"/>
+    <link href="https://www.e-elgar.com/sites/e-elgar/files/images/shared/authoredguidebussocsci20151.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <summary>Style for Edward Elgar Publishing Business and Social Sciences (non-textbook) publications.</summary>
+    <updated>2017-02-03T13:40:05+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="and others">""</term>
```

Comment:
@damnation333, see http://docs.citationstyles.org/en/stable/specification.html#et-al. The "and others" term is only used when set explicitly on `<et-al/>`.

(and, like @bwiernik says, some kind of "et al." term should always be used when abbreviating name lists, since otherwise there is no indication that names were left out)